requirements.txt

```diff
@@ -1,2 +1,2 @@
-honeybee-core==1.39.4
+lbt-honeybee==0.2.221
```

Comment:
You should update this before merging.

Reply:
Following is what I have tin requirements;
lbt-honeybee==0.2.221
rhino3dm==0.13.0
I am not sure if I need to update something here.

Reply:
The current version of `lbt-honeybee` is 0.2.249:
https://github.com/ladybug-tools/lbt-honeybee

The version gets incremented with every change that we merge for honeybee or ladybug so you should probably check it right before your merge.

Reply:
Okay. I will keep this in mind when we merge.